src/prediction.jl

```diff
@@ -336,4 +147,86 @@ function unstack_techreplicates(raw_data, retain_first=true)
     unstack(raw_data, :variable, :value; allowduplicates=true)
 
     return unstackedDf
-end #end function
\ No newline at end of file
+end #end function
+
+function report_classification_merit(classification_results::Dict)
+    classification_merits = DataFrame(
+    :Trial => collect(1:classification_results[:n_trials]),
+    :Train_ACC => classification_results[:train_accuracies],
+    :Test_ACC => classification_results[:test_accuracies],
+    )
+
+    push!(classification_merits, Dict(
+        :Trial => 0,
+        :Train_ACC => mean(classification_merits.Train_ACC),
+        :Test_ACC => mean(classification_merits.Test_ACC)
+    ))
+
+    return classification_merits
+end
+
+function report_regression_merit(regression_results::Dict)
+    regression_merits = DataFrame(
+        :Trial => collect(1:regression_results[:n_trials]),
+        :Train_MAE => regression_results[:train_maes],
+        :Test_MAE => regression_results[:test_maes],
+        :Train_MAPE => regression_results[:train_mapes],
+        :Test_MAPE => regression_results[:test_mapes],
+        :Train_COR => regression_results[:train_correlations],
+        :Test_COR => regression_results[:test_correlations]
+        )
+
+    push!(regression_merits, Dict(
+        :Trial => 0,
+        :Train_MAE => mean(regression_merits.Train_MAE),
+        :Test_MAE => mean(regression_merits.Test_MAE),
+        :Train_MAPE => mean(regression_merits.Train_MAPE),
+        :Test_MAPE => mean(regression_merits.Test_MAPE),
+        :Train_COR => mean(regression_merits.Train_COR),
+        :Test_COR => mean(regression_merits.Test_COR)
+    ))
+    
+    return regression_merits
```

Comment:
I think using a NamedTuple here is *slightly* more performent than a Dict - so just

```suggestion
    push!(regression_merits, (;
        Trial = 0,
        Train_MAE = mean(regression_merits.Train_MAE),
        Test_MAE = mean(regression_merits.Test_MAE),
        Train_MAPE = mean(regression_merits.Train_MAPE),
        Test_MAPE = mean(regression_merits.Test_MAPE),
        Train_COR = mean(regression_merits.Train_COR),
        Test_COR = mean(regression_merits.Test_COR)
    ))
    
    return regression_merits
```